src/app/modules/main/wallet_section/accounts/compact_item.nim

```diff
@@ -0,0 +1,89 @@
+import strformat
+
+type
+  Item* = object
```

Comment:
don't we already have a similar item and model?

Reply:
Hey @alaibe yes we do ! this is like just a reduced version of the account `item.nim` and `model.nim`. I need this list of `related accounts within an account item` and it was simpler to just create this.